site/content/en/docs/Guides/metrics.md

```diff
@@ -184,7 +184,7 @@ Grafana is a open source time series analytics platform which supports Prometheu
 First we will install [Agones dashboard](#grafana-dashboards) as [config maps](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/) in our cluster.
 
 ```bash
-kubectl apply -f ../build/grafana/
+kubectl apply -f ./build/grafana/
```

Comment:
This change was done to comply with prometheus installation step, so all command are performed from Agones repo root.